core/storage/classifier/gae_models.py

```diff
@@ -42,7 +38,7 @@ class ClassifierTrainingJobModel(base_models.BaseModel):
 
     # The ID of the algorithm used to create the model.
     algorithm_id = datastore_services.StringProperty(
-        required=True, choices=ALGORITHM_CHOICES, indexed=True)
+        required=True, indexed=True)
```

Comment:
Why this change?

Reply:
Good catch. This was required for some unit tests in earlier design where we were actually reading the trained model from datastore instead of its name. Later we decided to just read name of the file which stores trained model and then the frontend will pass us this name to retrieve the actual model. Hence, this can be undone for now but maybe we will need in the next PR where the actual reading will happen.

Reply:
My bad. ALGORITHM_CHOICES have to removed for certain unit tests to work which validate that errors are raised if algorithm id is not found. This needs to be kept as it is.

Reply:
Ignoring this.